addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -4,6 +4,18 @@ import { CalendarModel } from "@web/views/calendar/calendar_model";
 import { askRecurrenceUpdatePolicy } from "@calendar/views/ask_recurrence_update_policy_hook";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
 
+function isSameDay(oldStartDate, oldEndDate, newStartDate, newEndDate) {
+    /**
+     * Auxiliary function for indicating if two dates are within the same day.
+     * Used in "All events" recurrences update type for ensuring that event updates are within the same day.
+     */
+    let oldStart = new Date(oldStartDate);
+    let oldEnd = new Date(oldEndDate);
+    let newStart = new Date(newStartDate);
+    let newEnd = new Date(newEndDate);
+    return oldStart.toDateString() == newStart.toDateString() && oldEnd.toDateString() == newEnd.toDateString();
```

Comment:
You can use the `areDatesEqual` function defined in `addons/web/static/src/core/l10n/dates.js`

Reply:
Ah, indeed. You can maybe use the hasSame method from luxon https://moment.github.io/luxon/#/math?id=comparing-datetimes

Reply:
Cool! Updating the code, thanks for sharing it 😄